content/bn/docs/concepts/configuration/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "কনফিগারেশন"
+weight: 80
+description: >
+  পডস কনফিগার করার জন্য কুবারনেটিস যে সংস্থানগুলি প্রদান করে ।
+---
+
```

Comment:
Don't add extra line please empty lines are also visible in website

Reply:
It shows Error in user "YAML: (<unknown>): could not find expected ':' while scanning a simple key at line 4 column 1 ! of I remove spaces" if I remove spaces  [image](https://github.com/kubernetes/website/assets/111334392/da960969-948e-4282-a248-e6a0abf42a45)